### PR TITLE
fix scalerank 0 region boundaries, drop low-zoom names, add map_unit

### PR DESCRIPTION
**Pull request by nvkelso**
connects to #992 to fix scalerank 0 region boundaries
also adds map_unit boundaries and
drops name at low zooms 0,1,2,3,4,5,6

- [x] Update tests
- [x] Update data download and config
- [x] Update docs

### CODE REVIEW

**Comment by nvkelso** [comment]
@zerebubuth Looks like my adding `map_unit` as a new Natural Earth data source is making all the tests fail. Can you take a look, please?

**Comment by zerebubuth** [comment]
The first error seems to be:

```
ERROR:  column "mapcolor9" of relation "ne_10m_admin_0_boundary_lines_map_units" does not exist
invalid command \N
invalid command \.
ERROR:  syntax error at or near "5.00000000000"
LINE 1: 5.00000000000 Map unit boundary 
        ^
ERROR:  current transaction is aborted, commands ignored until end of transaction block
```

Which is preventing the shapefiles for the map units being loaded. It looks like the [schema](https://github.com/tilezen/vector-datasource/blob/fae00e841d1497cd7dd72396a8728de134b30cf2/data/shapefile_schema/ne_10m_admin_0_boundary_lines_map_units.sql) doesn't contain a column called !mapcolor9", but the data does. We don't currently support loading a subset of the columns - they must all be specified in the schema.

The second error seems to be:

```
psql:apply-updates-non-planet-tables.sql:107: ERROR:  column "scalerank" does not exist
LINE 15: ...recla" = 'Admin-1 region boundary' THEN CASE WHEN scalerank ...
```

Which probably means that `extra_columns: scalerank` needs to be specified when using the global [`ne_region_boundaries_min_zoom`](https://github.com/tilezen/vector-datasource/pull/1037/files#diff-dca2f4be5bc54303506aa7848a476050R10).

**Comment by nvkelso** [comment]
Thanks! Fixed in b39f38bb911dd40b3081af9dc291fd22ecb51117 and 26e1c1562c59d22fc050abb41c96d6f751409c8f, waiting for CirlceCI to confirm.

File: yaml/boundaries.yaml [hunk 1]
```diff
@@ -75,3 +80,21 @@ filters:
...
+    table: ne
+  - filter:
+      featurecla:
+        - Admin-1 boundary
+        - Admin-1 statistical boundary
+        - Admin-1 statistical meta bounds
+    min_zoom: *ne_region_boundaries_min_zoom
+    extra_columns: [scalerank]
+    output: {kind: region, kind_detail: '4'}
+    table: ne
```
Comment (rmarianski) [inline yaml/boundaries.yaml]: no trailing newline

**Comment by rmarianski** [comment]
LGTM

File: yaml/boundaries.yaml (hunk 1 above)
Comment (nvkelso) [inline yaml/boundaries.yaml]: Fixed in ad617c94a7e25f1154aabcbeaef6e787e9140e08.

**Comment by rmarianski** [comment]
I checked in some fixes. We'll see if the tests pass on circle now :fingers-crossed:

**Comment by zerebubuth** [comment]
> We'll see if the tests pass on circle now

Do they pass locally? :stuck_out_tongue_winking_eye:

**Comment by rmarianski** [comment]
> Do they pass locally? :stuck_out_tongue_winking_eye:

I'm going to wait until the circle build finishes before answering that :stuck_out_tongue_closed_eyes:

**Comment by nvkelso** [comment]
Our current thinking is the data files here need to be in web Mercator (but are probably in geographic now).

**Comment by nvkelso** [comment]
Last test failures are related to OSM zoom 8 content, what gives?

**Comment by zerebubuth** [comment]
Boundaries looked like they just needed the names swapped, possibly because the direction was reversed due to the introduction of stable ordering in #1022.

The second failure looks like it was also a copy-paste issue with the tile coordinates.

Works for me now, let's see if CircleCI agrees :wink:

**Comment by nvkelso** [comment]
Woot! Merging.

**Pull request merged**